webapp/babel.config.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const config = {
+    presets: [
+        ['@babel/preset-env', {
+            targets: {
+                chrome: 66,
+                firefox: 60,
+                edge: 42,
+                safari: 12,
```

Comment:
Have we tested the UI on these browsers?

Reply:
@AayushChaudhary0001

Reply:
We will be testing it on firefox.